application/controllers/admin/Tokens.php

```diff
@@ -1438,7 +1441,16 @@ public function email(int $iSurveyId)
                                 $oToken->remindercount++;
                             }
                             $tokenSaveError = "";
-                            if (!$oToken->encryptSave(true)) {
+                            if (
```

Comment:
No encryption needed?

Reply:
We save only not encrypted attributes, then no encryption needed :) 

Interesting because we don't update other attribute (we don't encrypt it)